augly/video/helpers/intensity.py

```diff
@@ -42,6 +43,15 @@ def audio_swap_intensity(offset: float, **kwargs) -> float:
     return (1.0 - offset) * 100.0
 
 
+def augment_audio_intensity(aug_function: str, **kwargs) -> float:
+    intensity_func = getattr(aud_intensity, f"{aug_function}_intensity", None)
+    """
+    says missing 1 positional argument but if I pass in aug_function 
+    then apply_lambda_intensity will become recursive
+     """
+    return intensity_func(**kwargs) if intensity_func else 100.0
```

Comment:
```suggestion
def augment_audio_intensity(audio_metadata: List[Dict[str, Any]], **kwargs) -> float:
    return audio_metadata[0]["intensity"]
```

We can get the intensity for the audio augmentation that was called from the metadata we got from calling the audio augmentation, since the intensity was already computed inside the audio augmentation when creating the audio metadata.